test/python/fx_importer/basic_test.py

```diff
@@ -88,12 +88,13 @@ def forward(self, x):
 
 @run
 # CHECK-LABEL: test_import_frozen_exported_program_with_dynamic_shapes
-# CHECK:     func.func @test_net(%[[ARG0:[a-zA-Z0-9]+]]: !torch.vtensor<[?,4],f32>) -> !torch.vtensor<[?,4],f32>
+# CHECK:     func.func @test_net(%[[ARG0:[a-zA-Z0-9]+]]: !torch.vtensor<[?,?,5],f32>) -> !torch.vtensor<[?,?,5],f32>
 # CHECK:     %[[S0:.*]] = torch.symbolic_int "s0" {min_val = {{[0-9]+}}, max_val = {{[0-9]+}}} : !torch.int
-# CHECK:     torch.bind_symbolic_shape %[[ARG0]], [%[[S0]]], affine_map<()[s0] -> (s0, 4)> : !torch.vtensor<[?,4],f32>
-# CHECK:     %[[TANH:.*]] = torch.aten.tanh %[[ARG0]] : !torch.vtensor<[?,4],f32> -> !torch.vtensor<[?,4],f32>
-# CHECK:     torch.bind_symbolic_shape %[[TANH]], [%[[S0]]], affine_map<()[s0] -> (s0, 4)> : !torch.vtensor<[?,4],f32>
-# CHECK:     return %[[TANH]] : !torch.vtensor<[?,4],f32>
+# CHECK:     %[[S1:.*]] = torch.symbolic_int "s1" {min_val = 2, max_val = 9223372036854775806} : !torch.int
```

Comment:
JFYI we kept `min_val` check open (`min_val = {{[0-9]+}}`) because when not set it is 0 with the nightlies and 2 with stable; this is short-lived - once we bump PT stable we can make it explicit `min_val = 0` everywhere. You're fine since you're setting it in for channel dim to 2.

Also, max_val should actually be `9223372036854775807` (off by 1) once you fix the bug above.